static/js/board_events.js

```diff
@@ -82,20 +83,36 @@ function createFeaturedEventObject(eventData) {
  * Time Unit : ms.  Default Settings : 10s, 7s(small slide)
  */
 $(function(){
+
+  // parse events from abe JSON URL
   $.get('https://abe-dev.herokuapp.com/events/', populateEvents);
-  //  // Initializes auto scroll for events
-   var scrolltopbottom = setInterval(function(){
+
+  // Initializes auto scroll for events
+   var eventsTodayScroll = setInterval(function(){
     $('.today .autoscrolling > .collection').animate({ scrollTop: $('.today .autoscrolling > .collection').prop('scrollHeight') }, 12000);
     setTimeout(function() {
-       $('.autoscrolling > .collection').animate({scrollTop:0}, 8000);
+       $('.today .autoscrolling > .collection').animate({scrollTop:0}, 12000);
     },4000);
-  },4000);
+  },2000);
 
-  //  // Initializes auto scroll for events
-   var scrolltopbottom = setInterval(function(){
-    $('.featured .autoscrolling > .collection').animate({ scrollTop: $('.featured .autoscrolling > .collection').prop('scrollHeight')  }, 10000);
+  // Initializes auto scroll for featured events
+   var eventsFeaturedScroll = setInterval(function(){
+    // scrolls to bottom over 12000ms
+    $('.featured .autoscrolling > .collection').animate({ scrollTop: $('.featured .autoscrolling > .collection').prop('scrollHeight')  }, 12000);
+
+    // scrolls back to top after 4000 ms over 12000ms
     setTimeout(function() {
-       $('.featured .autoscrolling > .collection').animate({scrollTop:0}, 8000);
+       $('.featured .autoscrolling > .collection').animate({scrollTop:0}, 12000);
```

Comment:
maybe another way we could do this with functions and set timeouts vs an interval. See the following for an example:
```
function scrollEvents($collectionSelection, scrollToBottom = true) {
  $collectionSelection.animate({ scrollTop: (scrollToBottom) ? $collectionSelection.prop('scrollHeight') : 0 }, 12000);
  setTimeout(function() {
    scrollEvents($collectionSelection, !scrollToBottom);
  }, 4000)
}
scrollEvents($('.featured .autoscrolling > .collection'), true);
```